Example/Auth/README.md

```diff
@@ -84,6 +103,18 @@ Please follow the instructions in
 [ApiTests/AuthCredentialsTemplate.h](ApiTests/AuthCredentialsTemplate.h)
 to generate the AuthCredentials.h file.
 
+#### Console
+
+In the Firebase conosle for your test project, you'll need to enable the
+following auth providers:
+* Email/Password
+* Google
+* Facebook
+* Anonymous
+
+You'll also need to create a user with email
+`iosgcip+email_existing_user@gmail.com` and password of `password`.
```

Comment:
Our testing code was not designed for public developers. We should use some place holder here.